src/main/java/com/RaiseTech8/MyBatisdemo/Name.java

```diff
@@ -0,0 +1,25 @@
+package com.RaiseTech8.MyBatisdemo;
+
+public class Name {
```

Comment:
クラス名よくないですね。
国の花を表す名前にするのがbetterです。
```suggestion
public class NationalFlower {
```

Reply:
9c781811f0d3004550bd3c0669a02047d18feb56

- クラス名をレコードに対応したものに変更しました。